framework/src/main/java/org/checkerframework/framework/util/dependenttypes/DependentTypesHelper.java

```diff
@@ -415,7 +401,36 @@ public void standardizeReturnType(MethodTree methodDeclTree, AnnotatedTypeMirror
         JavaExpressionContext context =
                 JavaExpressionContext.buildContextForMethodDeclaration(
                         methodDeclTree, pathToMethodDecl, factory.getChecker());
-        standardizeDoNotUseLocalScope(context, pathToMethodDecl, atm);
+        viewpointAdaptToContext(context, atm);
+    }
+
+    /**
+     * Viewpoint-adapt a type qualifier obtained from a contract to {@code jeContext}.
+     *
+     * @param annoFromContract a qualifier that was written on a method declaration
+     * @param jeContext the context to use for standardization
```

Comment:
I thought contexts were used for viewpoint-adaptation (and this method is a viewpoint-adaptation method).